x/mongo/driver/driver.go

```diff
@@ -75,6 +73,23 @@ type Connection interface {
 	Stale() bool
 }
 
+// RTTMonitor represents a round-trip-time monitor.
+type RTTMonitor interface {
+	// AvgRTT returns the exponentially weighted moving average observed round-trip time.
+	AvgRTT() time.Duration
```

Comment:
Optional: Consider renaming this to `EWMA` or similar to indicate it's not the average of the samples used for the other calculations.